File: app/src/main/java/com/example/samson/immvrse/MainActivity.java
package com.example.samson.immvrse;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.design.widget.NavigationView;
import android.support.v4.app.FragmentActivity;
import android.view.Gravity;
import android.view.MenuItem;
import android.view.View;
import android.widget.Toast;

import com.example.samson.immvrse.databinding.ActivityMainBinding;
import com.example.samson.immvrse.news_feed.NewsFeedFragment;


public class MainActivity extends FragmentActivity {

    ActivityMainBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_main);
        binding.menuButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                binding.drawerLayout.openDrawer(Gravity.START);
            }
        });

        getSupportFragmentManager().beginTransaction()
                .add(R.id.content_container, new NewsFeedFragment())
                .commit();

        binding.navDrawer.setItemIconTintList(null);
        binding.navDrawer.setNavigationItemSelectedListener(new NavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                Toast.makeText(getApplicationContext(), "Clicked " + item.getTitle(), Toast.LENGTH_SHORT).show();
                return true;
            }
        });
    }

}

File: app/src/main/java/com/example/samson/immvrse/MainActivity.kt
package com.example.samson.immvrse

import android.os.Bundle
import android.support.v4.app.FragmentActivity
import android.view.Gravity
import android.widget.Toast
import com.example.samson.immvrse.news_feed.NewsFeedFragment
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        menu_button.setOnClickListener {drawer_layout.openDrawer(Gravity.START) }

        supportFragmentManager.beginTransaction()
                .add(R.id.content_container, NewsFeedFragment())
                .commit()

        nav_drawer.itemIconTintList = null
        nav_drawer.setNavigationItemSelectedListener { item ->
            Toast.makeText(applicationContext, "Clicked " + item.title, Toast.LENGTH_SHORT).show()
            true
        }
    }
}


